relay-event-schema/src/protocol/contexts/trace.rs

```diff
@@ -57,7 +58,17 @@ impl fmt::Display for TraceId {
 
 impl fmt::Debug for TraceId {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "TraceId(\"{}\")", self.0.as_simple())
```

Comment:
This was on purpose, to keep the debug representation short.

Reply:
Ok, in that case I'll change it back.